core/camel-core-engine/src/main/docs/modules/eips/pages/choice-eip.adoc

```diff
@@ -40,14 +40,49 @@ RouteBuilder builder = new RouteBuilder() {
             .choice()
                 .when(simple("${header.foo} == 'bar'"))
                     .to("direct:b")
+                .endChoice()
                 .when(simple("${header.foo} == 'cheese'"))
                     .to("direct:c")
+                .endChoice()
```

Comment:
Can you restore this example back to what it was before - this is 99% of the normal use-cases.

And then below it would be good to add a header section about endChoice and then you can use the multicast example as its example.

Reply:
Hi Dave,
Sure Dave, I have made changes. Let me know if they are correct. I am happy to make any additional changes that you can share. :+1:

I have done some minor reordering of documentation, let me know if that's okay. (TIP is moved from being in between dsl and xml to end of doc)

Reply:
Thanks for your patience and helping with the project. You are much welcome to keep contributing, thanks.